### PR TITLE
Issue 06 seir model

### LINKED ISSUE #6: Implement SEIR model class
Implement SEIR model class specified in [here](https://benlambertdotcom.files.wordpress.com/2020/10/week_project.docx).

- [x] Define the SEIR model class in the `models.py` called `SEIRModel` that inherits from #5.
- [x] Implement the simulate method for the SEIR model with inputs and outputs according to the base class doc string. Hint: You can choose any numerical integration package that you like.
- [x] Write a doc string which explains and defines the differential equations, see [here](https://github.com/pints-team/pints/blob/master/pints/toy/_fitzhugh_nagumo_model.py) for the raw doc string, and [here](https://pints.readthedocs.io/en/stable/toy/fitzhugh_nagumo_model.html) for the sphinx illustration.

### PR DESCRIPTION
**Pull request by siting-miao**
Closes #6

### CODE REVIEW

**Comment by DavAug** [comment]
@siting-miao 
Could you add a file to the root directory called `.coveragerc` with the follwoing content
```
  
[run]
source = seirmo
omit = 
    seirmo/tests/*

[report]
exclude_lines =
    pragma: no cover
    pragma: no linux cover
    pragma: no python 3 cover
    def __repr__
    raise AssertionError
    raise NotImplementedError
    except ImportError:
    if False:
    if __name__ == .__main__.:
```

**Comment by DavAug** [comment]
This just makes sure codecove is not looking at the tests for coverage.

**Comment by siting-miao** [comment]
> This just makes sure codecove is not looking at the tests for coverage.

Thx! Done with that. I'm now going to do the docstring test issue?

**Comment by siting-miao** [comment]
> This just makes sure codecove is not looking at the tests for coverage.

Ah I see there have been several issues created. I'll go to them later in the day since there is a Career Development session now.

File: .coveragerc [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[run]
+source = seirmo
+omit = 
+    seirmo/tests/*
```
Comment (DavAug) [inline .coveragerc:4]: Could you also add the version_info.py here, i.e.
```
omit = 
    seirmo/test/*
    seirmo/version_info.py
```

**Review by DavAug — CHANGES_REQUESTED** [review]
This looks really really good now, thank you @siting-miao !

I just have two minor things and then I am happy to merge this in:
- [x] Could you also ignore the version_info.oy for the coverage (as indicated in the comment)
- [x] Could you remove the line in the .flake8 that allows you to have blank line at the end of the file? Having a blank line at the end of the file just looks a lot nicer, so it would be good if flake8 would remind us of that. This basically boils down to removing this `W391` from the ignore.

File: .coveragerc (hunk 1 above)
Comment (siting-miao) [reply .coveragerc:4]: I have done those:) Thank you!

**Review by DavAug — APPROVED** [review]
That's really good! Thank you @siting-miao

**Pull request merged**